docs/en/getting-started/install.md

```diff
@@ -110,25 +110,6 @@ sudo service clickhouse-server start
 clickhouse-client # or "clickhouse-client --password" if you've set up a password.
 ```
 
-<details>
-<summary>Deprecated Method for installing deb-packages</summary>
-
-``` bash
-sudo apt-get install apt-transport-https ca-certificates dirmngr
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E0C56BD4
-
-echo "deb https://repo.clickhouse.com/deb/stable/ main/" | sudo tee \
-    /etc/apt/sources.list.d/clickhouse.list
-sudo apt-get update
-
-sudo apt-get install -y clickhouse-server clickhouse-client
-
-sudo service clickhouse-server start
-clickhouse-client # or "clickhouse-client --password" if you set up a password.
-```
-
-</details>
-
```

Comment:
Should we also get rid of section `Migration method for installing the deb-packages`? (l. 133ff)

Reply:
we can call it a new deprecated =)

It was a classic way to install keys, and old distros can still use it. Like, 14.04 or so

Reply:
Done